contents/blog/measure-product-market-fit.md

```diff
@@ -0,0 +1,395 @@
+
+## Indicator #1: Word-of-mouth growth
+
+- **Type:** Leading indicator
+- **Good for:** Product-led companies
+
+You _can't_ validate product-market fit _using word of mouth alone_. That way lies madness. But it is a useful indicator _when confirmed by other metrics_, such as user engagement.
+
+Hopefully, you already have a strong grasp of your organic user growth via user signups, or whatever metric makes sense for your product. But you can, with a little work, also track word-of-mouth awareness and sentiment of your product. Here are a few options:
+
+### 1. Brand mention alerts
+
+Use a tool like [Syften](https://syften.com/) to monitor social media, communities, and newsletters for people talking about you. We use it at PostHog to send alerts to a `#brand-alerts` channel in our Slack. 
+
+Tracking brand mentions is more about understanding what people are saying about you, rather than tracking a trend. People spontaneously sharing their love for your product is a good sign. Identify who these people are and why your product is such a good fit for them.
+
+> **❗️Important:** It's _much easier_ to track mentions if your company has a unique name. You're bang out of luck if it's a common noun – e.g. Apple, Amplitude. 
+
+### 2. Searches for your brand
+
+**Google Trends** is a good option if you have a unique brand name. Just plug your brand into Trends to see how searches for your brand are trending over time.
+
+![google trend for posthog](../images/blog/how-to-measure-product-market-fit/posthog-trends.png)
+
+It's also useful for comparing yourself to other companies, such as those you _know_ have product-market fit or competitors.
+
+![google trend comparison](../images/blog/how-to-measure-product-market-fit/posthog-with-logrocket.png)
+
+At PostHog, we also use [Glimpse](https://meetglimpse.com/) – an extension that augments Trends by adding trend lines, forecasts, and real search volumes. 
+
+![trends with Glimpse](../images/blog/how-to-measure-product-market-fit/glimpse.png)
+
+The above shows how PostHog showed signs of product-market fit in late 2021 / early 2022, which grew stronger from mid-2022 onwards. While exponential growth is desirable, linear growth is still a good sign.
+
+Glimpse also suggests similar companies – useful for comparing your growth with those you know have product-market fit.
+
+![related companies](../images/blog/how-to-measure-product-market-fit/related-companies.png)
+
+### 3. Organic traffic to your homepage
+
+You can verify the above using organic visitors to your website homepage as a proxy for word of mouth. We track this in PostHog as an insight configured to show:
+
+1. Unique users who visited the homepage via a search engine.
+2. Unique users who visited the homepage directly.
+3. While excluding users with an identified email address.
+
+This captures new users who visit our homepage by searching for us, typing in our website address, or via "dark social" sources like Slack and WhatsApp.
```

Comment:
```suggestion
This captures new users who visit our homepage by searching for us, typing in our website address, or via dark social sources like Slack and WhatsApp.
```
It's a nit, but I don't generally like quotations like this as they can come across as snarky.